pokemongo_bot/tree_config_builder.py

```diff
@@ -31,13 +31,16 @@ def build(self):
 
         for task in self.tasks_raw:
             task_type = task.get('type', None)
-            if task_type is None:
-                raise ConfigException('No type found for given task {}'.format(task))
-            elif task_type == 'EvolveAll':
-                raise ConfigException('The EvolveAll task has been renamed to EvolvePokemon')
-
             task_config = task.get('config', {})
 
+            # Validate task name refactors
+            if type is None:
+                raise ConfigException('No type found for given task {}'.format(type))
+            elif type == 'EvolveAll':
```

Comment:
After posting about the "if type is None" line, I checked the changes and I mightve probably confused about task_type and type vars, although I dont get any errors, but I'll look at it.